docs/notebooks/quickstart.ipynb

```diff
@@ -611,11 +611,11 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.12"
+   "version": "3.10.6"
```

Comment:
Just a general comment: we strive to support all versions of python from 3.8 onward.

Reply:
I think this is just auto-changed by the conda environment I was running in (and ditto whoever ran it before me). Not sure how to force it to stay the same on Github.